contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -286,13 +286,23 @@ SELECT pg_catalog.pg_extension_config_dump('sys.babelfish_configurations', '');
 -- SERVER_PRINCIPALS
 CREATE OR REPLACE VIEW sys.server_principals
 AS SELECT
-CAST(Base.rolname AS sys.SYSNAME) AS name,
+CAST(
+  CASE
+    WHEN Ext.orig_loginname IS NOT NULL THEN Ext.orig_loginname
```

Comment:
orig_loginname column will never be NULL right? As we will be filling that with rolname if it is empty. So, why are we doing this?

Reply:
Good point! updated the definition